pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/MultiStageEngineIntegrationTest.java

```diff
@@ -1078,6 +1078,22 @@ public void testFilteredAggregationWithNoValueMatchingAggregationFilterWithOptio
     assertEquals(result.get("numRowsResultSet").asInt(), 0);
   }
 
+  @Test
+  public void testWindowFunction()
```

Comment:
This test is added to check if queries with these window functions execute ?

Reply:
Yeah basically to verify the end to end flow (query planning + execution with runtime operators) works without errors.